docs/configuration.md

```diff
@@ -41,40 +41,20 @@ Broker service configuration values:
 | <tt>SECURITY_USER_NAME</tt> <b>*</b> | api.user | string | <p>Broker authentication username</p>|
 | <tt>SECURITY_USER_PASSWORD</tt> <b>*</b> | api.password | string | <p>Broker authentication password</p>|
 | <tt>PORT</tt> | api.port | string | <p>Port to bind broker to</p>|
+| <tt>HOST</tt> | api.host | string | <p>Host to bind broker to</p>|
```

Comment:
**question**: My apologies in advance if the question make no sense or does not apply to the context of the modification. There is a file `script/set-env.sh` where we define the environment variables `SECURITY_USER_NAME`, `SECURITY_USER_PASSWORD` and `PORT` among others. I do not see the `HOST` variable and although it is out of the context of this Pull Request, I wondered if you think it is necessary to add the new variable you have added to the documentation, or if it will be added in another PR, or simply the variable does not have to be added at all.

If we finally have to add this variable to the file, we may also have to add it to `Makefile` to run the app.

Reply:
Good shout @zucchinidev. I am just adding it to the docs because `HOST` is indeed configurable. It does default to `localhost` though (whereas `PORT` doesn't have a default), so there's no need to add it to the `set-env.sh` script.  The `makefile` could be improved in many ways, and we could change it to make use of all the configured variables (at the moment even `PORT` is hardcoded and `HOST` not passed as it has a sensible default).  Just the need to configure those two variables hasn't come up.

Reply:
Fantastic! thank you very much for the answer. All clear to me!